<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEnch</class>
 <widget class="QDialog" name="AddEnch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AddEnchantment</string>
  </property>
  <widget class="QLabel" name="lbITEM">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Item:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbItmID">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(ItemID)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddEnch">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>451</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Enchantment Settings</string>
   </property>
   <widget class="QComboBox" name="cbbEnchantments">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Protection</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Feather Falling</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fire Protection</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blast Protection</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Projectile Protection</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Respiration</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aqua Affinity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thorns</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Depth Strider</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Frost Walker</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Curse of Binding</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sharpness</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Smite</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bane of Arthropods</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Knockback</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fire Aspect</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Looting</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sweeping Edge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Efficiency</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Silk Touch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Unbreaking</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fortune</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Power</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Punch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Flame</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Infinity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Luck of the Sea</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lure</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Loyalty(1.13+)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Impaling(1.13+)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Riptide(1.13+)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Channeling(1.13+)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mending</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Curse of Vanishing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Multishot(1.14+)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Piercing(1.14+)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quick Charge(1.14+)</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spbLvl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbEnch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enchantment</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbLvl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Level</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddEnch</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
